source/tests/system/nidmm_driver_api_tests.cpp

```diff
@@ -145,6 +145,58 @@ class NiDmmDriverApiTest : public ::testing::Test {
             return response.attribute_value();
         }
 
+        void configure_current_source(double value)
+        {
+            ::grpc::ClientContext context;
```

Comment:
Some methods have two-space indents while some have four. Can you update them all to have two spaces per indent level?

Reply:
fixed this